API/controlsClass/controlsClass.m

```diff
@@ -664,6 +624,19 @@
             obj.display = inputBlock.display;
         end
 
+        function inputBlock = parseInputs(obj, p, properties, errorMsg)
```

Comment:
I'd make this routine private (i.e., include in a `methods(Access = private)` block, since it refers only to the routines defined above and will not be used outside of the class.

Reply:
The parsing methods have been made private.